Add arm64 in snap release

antlassagne opened the pull request.

IsmaelMartinez commented on package.json:
```diff
@@ -34,6 +34,7 @@
     "dist:linux:appimage": "electron-builder --x64 --armv7l --arm64 -l AppImage",
     "dist:linux:snap": "electron-builder -l snap",
     "dist:linux:snap:armv7l": "electron-builder --armv7l -l snap",
+    "dist:linux:snap:arm64": "electron-builder --arm64 -l snap",
```
no, as that will slow the release process.

antlassagne replied:
No to gemini, or no to me? :)

IsmaelMartinez replied:
Gemini, AI is helpful but sometimes doesn't have the whole/right context

IsmaelMartinez commented:
if you follow the release notes in the docs, this can make it in a new release. Just increase the lower number. Thanks! 2.5.11 I think

IsmaelMartinez commented on package.json:
```diff
@@ -49,6 +50,7 @@
     "electron-store": "8.2.0",
     "electron-window-state": "5.0.3",
     "lodash": "^4.17.21",
+    "n": "^10.2.0",
```
Can you remove this module? is not related to the app. Their docs do recommend installing it globally, not locally. Thanks!

(I use nvm but I am not curious how it compares :) )

IsmaelMartinez commented:
oh, looks like 1 part is missing. Can you run `npm i` as otherwise the package-lock.json file is out of sync with the version? Other than that is ready for release. Thanks!

antlassagne commented:
I rebased with main, updated the version and release note again, and package-lock.json.

I'm afraid I did not understand what part was missing, so it's probably still missing. What should I check?

IsmaelMartinez commented:
ok, something didn't work:

<img width="1026" height="301" alt="image" src="https://github.com/user-attachments/assets/4bc7edfa-c074-423d-bd13-075b2f21b58f" />

I think there is a typo in the snap file. I will check tomorrow if I can create a small release with the fix. It should be `dist:linux:snap:arm64` instead of `dist:linux:snap:armv64` (or changing the command in the package.json file.
